community-operators/rook/rook.v0.1.0.csv.openshift.yaml

```diff
@@ -0,0 +1,317 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Same comments for the other CSV. Also, the OpenShift CSV should be in the `community-operators/rook/` directory and the k8s CSV should be in the `upstream-community-operators/rook/` directory. Each directory will contain copies of CRDs and the package.yaml.